toolkit/scripts/toolchain/container/Dockerfile

```diff
@@ -66,6 +66,7 @@ COPY [ "./toolchain-md5sums", \
 WORKDIR $LFS/sources
 RUN wget -nv --no-clobber --timeout=30 --no-check-certificate --continue --input-file=$LFS/tools/toolchain-local-wget-list --directory-prefix=$LFS/sources; exit 0
 RUN wget -nv --no-clobber --timeout=30 --continue --input-file=$LFS/tools/toolchain-remote-wget-list --directory-prefix=$LFS/sources; exit 0
+RUN wget -nv --no-clobber --timeout=30 --continue https://github.com/microsoft/mariner-linux-kernel/archive/rolling-lts/mariner/5.10.13.1.tar.gz -O kernel-5.10.13.1.tar.gz --directory-prefix=$LFS/sources; exit 0
```

Comment:
Currently this link 404s for me since the repo isn't public yet, and it breaks the cgmanifest check as well- will the repo go live before you merge this change?

Reply:
Yes. That repo should go public before merging this PR.
I wanted to open up the current changes for PR review so I can incorporate any initial feedback.